### PR TITLE
Update README.md for v6

### PR DESCRIPTION
**Pull request by dannyhw**
Issue: #202 

## What I did

updated documentation for v6

## How to test

Please explain how to test your changes and consider the following questions

- read through documentation to see if it makes sense
- you might find it easier to read from the branch directly here https://github.com/storybookjs/react-native/tree/docs/v6-documentation

### CODE REVIEW

File: MANUAL_SETUP.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+- [expo setup script](https://gist.github.com/dannyhw/92b3ff0d6ccaead9df2820a507154b87)
+
+# Manual setup
+
+You may wish to setup everything yourself, you can use the following guide to do so.
+
+# Install dependencies
+
+**React native CLI**
```
Comment (flexbox) [inline MANUAL_SETUP.md]: Do you think you can follow the same order as Prebuilt?

**Prebuilt**
- expo
- react-native cli

**install deps**
- react-native cli
- expo

I would swap to 

**install deps**
- expo
- react-native cli

File: MANUAL_SETUP.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+cd ios; pod install; cd ..;
+```
+
+# Configuration
+
+## .storybook
+
+Create a folder called .storybook with files: main.js, preview.js, Storybook.tsx
+
+
```
Comment (flexbox) [inline MANUAL_SETUP.md:60]: I think we can automate with this one-liner

```suggestion
```console
mkdir .storybook && cd .storybook && touch main.js preview.js Storybook.tsx
```

```

File: MANUAL_SETUP.md (hunk 1 above)
Comment (dannyhw) [reply MANUAL_SETUP.md]: That's a good point, makes it more consistent. Thanks!

File: MANUAL_SETUP.md (hunk 2 above)
Comment (dannyhw) [reply MANUAL_SETUP.md:60]: Added!

**Pull request merged**